part-spec/power.json

```diff
@@ -0,0 +1,190 @@
+                    "comment": "units of seconds",
+                    "$ref": "definitions.json#/unit"
+                },
+                "integratedFets": {
+                    "description": "whether the regulator contains integrated switching mosfets",
+                    "type": "boolean"
+                },
+                "integratedFetProperties": {
+                    "description": "describes integrated fet current limits and rdson properties",
+                    "$ref": "#/$defs/powerFetProperities"
+                },
+                "undervoltageLockoutTresholdRising": {
+                    "description": "Undervoltage Lockout Out (UVLO) Threshold with input voltage rising",
+                    "comment": "units of Volts",
+                    "$ref": "definitions.json#/unit"
+                },
+                "undervoltageLockoutTresholdFalling": {
+                    "description": "Undervoltage Lockout Out (UVLO) Threshold with input voltage falling",
+                    "comment": "units of Volts",
+                    "$ref": "definitions.json#/unit"
+                },
+                "overvoltageProtectionTresholdRising": {
+                    "description": "Overvoltage Protection (OVP) Threshold with input voltage rising",
+                    "comment": "units of Volts",
+                    "$ref": "definitions.json#/unit"
+                },
+                "overvoltageProtectionTresholdFalling": {
+                    "description": "Overvoltage Protection (OVP) Threshold with input voltage falling",
+                    "comment": "units of Volts",
+                    "$ref": "definitions.json#/unit"
+                },
+                "thermalShudownRising": {
+                    "description": "Thermal Shudown (tsd) Threshold with temperature rising",
+                    "comment": "units of celsius",
+                    "$ref": "definitions.json#/unit"
+                },
+                "thermalShudownFalling": {
+                    "description": "Thermal Shudown (tsd) Threshold with temperature falling",
+                    "comment": "units of celsius",
+                    "$ref": "definitions.json#/unit"
+                },
+                "thermalShudownHysteresis": {
+                    "description": "Thermal Shutdown (tsd) Hysteresis",
+                    "comment": "units of celsius",
+                    "$ref": "definitions.json#/unit"
+                },
+                "efficiency": {
+                    "description": "power efficiency of regulator",
+                    "$ref": "graph.json#/graphDefiniton"
+                },
+                "pins": {
+                    "description": "array of pin objects with associated properties",
+                    "type": "array",
+                    "items": {
+                        "$ref": "definitions.json#/pinSpec"
+                    }
+                }
```

Comment:
The complianceList property is missing. This is the array to list the compliances met bt the component ( ROHS, AEC,...)

Reply:
This is now part of the ComponentID because it applies to all parts.